pkg/converter/convert_unix.go

```diff
@@ -601,6 +606,18 @@ func Merge(ctx context.Context, layers []Layer, dest io.Writer, opt MergeOption)
 					return errors.Wrap(err, "unpack nydus tar")
 				}
 
+				blobMetaPath := getBlobMetaPath(idx, ".blob.meta")
+				blobMetaFile, err := os.Create(blobMetaPath)
+				if err != nil {
+					return errors.Wrap(err, "create blob.meta file")
+				}
+
+				if _, err := UnpackEntry(layers[idx].ReaderAt, EntryBlobMeta, blobMetaFile); err != nil {
```

Comment:
Should continue decompressing the content part (not header part) for blobMetaFile.